src/manimShell.ts

```diff
@@ -75,8 +75,13 @@ export interface CommandExecutionEventHandler {
      * Callback that is invoked when the command is issued, i.e. sent to the
      * terminal. At this point, the command is probably not yet finished
      * executing.
+     * 
+     * @param shellStillExists Whether the shell still exists after the command
+     * was issued. In certain scenarios (e.g. user manually exits the shell
+     * during Manim startup), the shell might not exist anymore after the
+     * command was issued.
      */
-    onCommandIssued?: () => void;
+    onCommandIssued?: (shellStillExists) => void;
```

Comment:
Parameter 'shellStillExists' implicitly has an 'any' type.ts(7006)

I fixed it with: 

```ts
    onCommandIssued?: (shellStillExists: any) => void;
```